hack/lib/build/environment.sh

```diff
@@ -11,23 +11,24 @@ function os::build::environment::create() {
   local release_image="${OS_BUILD_ENV_IMAGE}"
   local additional_context="${OS_BUILD_ENV_DOCKER_ARGS:-}"
   if [[ "${OS_BUILD_ENV_USE_DOCKER:-y}" == "y" ]]; then
-    additional_context+="--privileged -v /var/run/docker.sock:/var/run/docker.sock"
+    additional_context+=" --privileged -v /var/run/docker.sock:/var/run/docker.sock"
+    local workingdir
+    workingdir=$( os::build::environment::release::workingdir )
```

Comment:
Previously the only way this was called was with a volume to "re-use" (which you created if you didn't have). The only way we wouldn't call this is if we were neither using local bind mount nor a volume ... which is when?